flaskr/templates/blog/update.html

```diff
@@ -16,4 +16,8 @@ <h1>{% block title %}Edit "{{ post['title'] }}"{% endblock %}</h1>
   <form action="{{ url_for('blog.delete', id=post['id']) }}" method="post">
     <input class="danger" type="submit" value="Delete" onclick="return confirm('Are you sure?');">
   </form>
+  <script src="https://cdn.ckeditor.com/4.16.0/standard/ckeditor.js"></script>
```

Comment:
I'm getting this error when I run this code:
This CKEditor 4.16.0 version is not secure. Consider [upgrading to the latest one](https://ckeditor.com/ckeditor-4-support/), 4.25.0-lts.